spec/Section 4 -- Introspection.md

```diff
@@ -132,7 +132,7 @@ type __Type {
   # should be non-null for OBJECT and INTERFACE only, must be null for the others
   fields(includeDeprecated: Boolean = false): [__Field!]
 
-  # should be non-null for OBJECT only, must be null for the others
+  # should be non-null for OBJECT and INTERFACE only, must be null for the others
```

Comment:
For my own clarity - this means implementations will start returning `[]` instead of `null` for existing interfaces which do not implement others?

Reply:
This is correct, simply to mirror the behavior of OBJECT types